packages/oas-form/src/utils.js

```diff
@@ -678,7 +678,7 @@ export function retrieveSchema(schema, rootSchema = {}, formData = {}) {
         }
       );
     } catch (e) {
-      console.warn(`could not merge subschemas in allOf:\n${e}`);
+      // console.warn(`could not merge subschemas in allOf:\n${e}`);
```

Comment:
not only weird that a console log made it through, but that we were testing it, thus expecting it.

Reply:
RJSF has a number of `console.warn`'s around that they're also asserting are getting thrown.